evaluate_iteration_changes.py

```diff
@@ -39,9 +40,22 @@ def collect_iteration_stats(
     sums_by_id = {}
     sum_ents_by_id = {}
     baseline_eval_sums_by_id = {}
+
+    if extrinsic_only:
+        all_extrinsic_ids = set(load_shuffled_test_split(xsum_test, 'bart-test-extrinsic', 'all').keys())
     for iteration_idx in sorted_keys:
         iteration_data = logs["iterations"][str(iteration_idx)]
         summaries = iteration_data["summaries"]
+
+        # filter full sets to be extrinsic only
+        if extrinsic_only:
```

Comment:
instead of needing to run GEF again to generate new log files, we can just filter those logs -- that's what I'm doing here